### ISSUE TITLE
[npm] load description/homepage from referenced node module version

### ISSUE THREAD

**Issue by JerryGreen** [microsoft/vscode#91086]
- VSCode Version: 1.42.1
- OS Version: macos Catalina 10.15.3

Steps to Reproduce:

1. `yarn add azer/indexeddb`
2. highlight `indexeddb` row in the `package.json

Does this issue occur when all extensions are disabled?: Yes

## Screenshot

<img width="556" alt="Screenshot 2020-02-20 at 18 41 42" src="https://user-images.githubusercontent.com/13215662/74938899-b6b8dd80-5410-11ea-8120-bed75f744742.png">

It shows https://github.com/bigeasy/indexeddb, though it should be  https://github.com/azer/indexeddb

**Comment by aeschli** [comment]
The URL shown is the package's homepage as stored in the NPM database.
We run `npm view indexeddb homepage` to get it.

**Comment by JerryGreen** [comment]
@aechan yes, but it's incorrect to always use key as the source (i.e. npm). When the value starts with "github:", the source is github, not npm. By running `npm install` you'll find it installs from github.

**Comment by aeschli** [comment]
If you want to make a PR that loads description / homepage from the referenced version, I'm happy to take it.

**Comment by JerryGreen** [comment]
It looks pretty straightforward, so yes, I'll likely look into it on the weekend or so. Might be a first PR into vscode, why not make one :)

I'll leave a pair of links of related code for later:

https://github.com/microsoft/vscode/blob/531522b76e6f0c170f6933d7f1967cade9384ad5/extensions/npm/src/features/packageJSONContribution.ts#L251-L257

https://github.com/microsoft/vscode/blob/531522b76e6f0c170f6933d7f1967cade9384ad5/extensions/npm/src/features/packageJSONContribution.ts#L260-L280
